csharp/Ice/async/Server.cs

```diff
@@ -9,44 +9,40 @@ public class Server
 {
     public static int Main(string[] args)
     {
-        int status = 0;
+        if (args.Length > 0)
```

Comment:
This is not correct. It's ok and common to start the server with for example `--Ice.Trace.Protocol=1`.